### PR TITLE
Formatting changes

### LINKED ISSUE #688: Fix GitHub Workflow formatting in Contributor's Guide
## Describe the Enhancement ##
The formatting of the GitHub Workflow in the Contributor's Guide does not look good:
https://dtcenter.github.io/METplus/develop/Contributors_Guide/github_workflow.html#github-workflow

### Time Estimate ###
1 day

### Sub-Issues ###
None

### Relevant Deadlines ###
None

### Funding Source ###
None

## Define the Metadata ##
None

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [x] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Review **projects** and select relevant **Repository** and **Organization** ones or add "alert:NEED PROJECT ASSIGNMENT" label
- [x] Select **milestone** to next major version milestone or "Future Versions"

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdatadb](https://github.com/dtcenter/METdatadb/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Enhancement Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [x] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, **Milestone**, and **Linked issues**
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by jprestop**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Compare current page: https://dtcenter.github.io/METplus/develop/Contributors_Guide/github_workflow.html#
with changes: https://ral.ucar.edu/staff/jpresto/work/METplus_jpresto/Contributors_Guide/github_workflow.html
to make the documentation more Sphinxified (not a word... but hopefully it conveys the thought).

- [x] Recommend testing for the reviewer to perform, including the location of input datasets:</br>
Compare current page: https://dtcenter.github.io/METplus/develop/Contributors_Guide/github_workflow.html#
with changes: https://ral.ucar.edu/staff/jpresto/work/METplus_jpresto/Contributors_Guide/github_workflow.html
to make the documentation more Sphinxified  (not a word... but hopefully it conveys the thought).

- [x] Will this PR result in changes to the test suite? **[Yes or No]**</br>
No

- [x] After merging, should the reviewer **DELETE** the feature branch from GitHub? **[Yes or No]**</br>
Yes

## Pull Request Checklist ##
See the [METplus Workflow](https://dtcenter.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**, **Project(s)**, and **Milestone**
- [x] After submitting the PR, select **Linked Issues** with the original issue number.

### CODE REVIEW

File: docs/Contributors_Guide/github_workflow.rst [hunk 1]
```diff
@@ -3,246 +3,349 @@
...
-| The branching model employed by the METplus GitHub repository is similar to that
-| described in the following document:
-|
-|      https://nvie.com/posts/a-successful-git-branching-model/
-|
-| where new or updated code is created on a 'feature' branch that is based on the NCAR/METplus GitHub 'develop' branch.
-| The feature branch is named after the corresponding GitHub issue:
-|       *feature_<Github Issue number>_<brief_description>*
+The branching model employed by the METplus GitHub repository is similar to
+thatdescribed in
```
Comment (georgemccabe) [inline docs/Contributors_Guide/github_workflow.rst]:
```suggestion
that described in
```
missing space!

File: docs/Contributors_Guide/github_workflow.rst [hunk 2]
```diff
@@ -3,246 +3,349 @@
...
 *Pre-requisite:*
 
-| Set up a GitHub repository account if you do not already have one, and log into your account.  For
-| more information about GitHub accounts, please refer to the following
-|
-|   https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account
-|
+Set up a GitHub repository account if you do not already have one, and log
+into your account.  For more information about GitHub accounts, please refer
+to the GitHub Documenation on `GitHub accounts <https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account>`_.
```
Comment (georgemccabe) [inline docs/Contributors_Guide/github_workflow.rst]:
```suggestion
to the GitHub Documentation on `GitHub accounts <https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account>`_.
```

File: docs/Contributors_Guide/github_workflow.rst [hunk 3]
```diff
@@ -3,246 +3,349 @@
...
+  where the asterisk, "*", indicates which branch is currently in use/checked out.
+
+
+Make changes to code in the feature branch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Users should make changes to their feature branch and commit changes to their
+local repository.
+
+* Create code following the coding standards in the Coding Standards section of
```
Comment (georgemccabe) [inline docs/Contributors_Guide/github_workflow.rst]: Consider creating a link to the Coding Standards section

File: docs/Contributors_Guide/github_workflow.rst [hunk 4]
```diff
@@ -3,246 +3,349 @@
...
+  .. code-block:: none
+		  
+    git pull upstream develop
+    git push origin develop
+
+  The first command pulls changes from the original repository (the
+  `METplus GitHub repository <https://github.com/dtcenter/METplus>`_ that you see when you
+  run *git remote -v* and that you set to upstream in step 4 above).  The second command
+  pushes those changes to your forked repository.
+
```
Comment (georgemccabe) [inline docs/Contributors_Guide/github_workflow.rst:230]: Consider adding instructions to merge the local develop branch into the feature branch, i.e.

`git merge origin develop`

File: docs/Contributors_Guide/github_workflow.rst [hunk 5]
```diff
@@ -3,246 +3,349 @@
...
+
+Commit changes to feature branch
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Commit any new files by running the following.  Perform this step only if this file is
```
Comment (georgemccabe) [inline docs/Contributors_Guide/github_workflow.rst:233]:
```suggestion
* Commit any new files by running the following.  Run the 'git add' command only if this file is
```

**Review by georgemccabe — APPROVED** [review]
The new formatting looks great! I added a few suggestions inline.

File: docs/Contributors_Guide/github_workflow.rst (hunk 3 above)
Comment (jprestop) [reply docs/Contributors_Guide/github_workflow.rst]: Thanks!  Done!  Still need to commit this change.

File: docs/Contributors_Guide/github_workflow.rst (hunk 4 above)
Comment (jprestop) [reply docs/Contributors_Guide/github_workflow.rst:230]: Done!

File: docs/Contributors_Guide/github_workflow.rst (hunk 3 above)
Comment (jprestop) [reply docs/Contributors_Guide/github_workflow.rst]: I have committed it.

**Comment by jprestop** [comment]
Hi George.  Thank for all of the suggestions.  I made all of them and checked them in.  Hopefully, they will be reflected here in this pull request.

**Comment by georgemccabe** [comment]
Changes look great, @jprestop . I made a few extra minor changes to make doc8 happy. The documentation job passed in Travis and the changes will not affect any other job, so I am going to merge the changes.

**Pull request merged**